tests/e2e/utils/service_utils.go

```diff
@@ -28,6 +28,10 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 )
 
+const (
+	serviceTimeout = 20 * time.Minute
```

Comment:
10 min to delete a service is not enough?

Reply:
it should work for deletion, but for creation, 10 minutes may be not working